File: benchmark/results/ts-script-sonnet/run-005/a-mcp/output/src/domain/notification/services/notification-service.ts
import { Notification } from '../aggregates/notification';

export class NotificationService {

  create(input: Partial<Notification>): Notification { throw new Error('not implemented'); }

  findById(id: string): Notification | null { throw new Error('not implemented'); }

  list(limit: number): Notification[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/notification/services/NotificationService.java
package com.metaengine.demo.notification.services;

import com.metaengine.demo.notification.aggregates.Notification;
import java.util.List;

/** Application service for the Notification aggregate. */
public class NotificationService {

    /** Create a new Notification from a partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Notification by id; returns null if not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Notifications, capped at the supplied limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Notification by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
